### PR TITLE
Choim nse7788 7789

### PR DESCRIPTION
**Pull request by choim-sci**
@covesturtevant , 

This PR has the unit test for def.wrte.avro.deve.R. 

Thanks,

### CODE REVIEW

File: pack/NEONprocIS.base/tests/testthat/test-def.wrte.avro.deve.R [hunk 1]
```diff
@@ -1,26 +1,143 @@
...
+                    NameSchm = NULL, 
+                    NameSpceSchm =  NULL, 
+                    Schm = NULL, 
+                    NameFileSchm = NULL, 
+                    NameLib = 'ravro.so')
+              expect_true (file.exists(outFile) &&avro_write_success == 0 ) 
+              # remove the test ouput file
+              if (file.exists(outFile)) {file.remove(outFile)}
+              # reset
+              writeSuccess = -1
```
Comment (covesturtevant) [inline pack/NEONprocIS.base/tests/testthat/test-def.wrte.avro.deve.R:88]: I think these writeSuccess instances should be avro_write_success. A few more below.

**Comment by covesturtevant** [comment]
@choim-sci Looks great other than a note I made above.

File: pack/NEONprocIS.base/tests/testthat/test-def.wrte.avro.deve.R (hunk 1 above)
Comment (choim-sci) [reply pack/NEONprocIS.base/tests/testthat/test-def.wrte.avro.deve.R:88]: Oh, a good catch.  
It was writeSuccess and somehow I introduced avro_write_success from my scratch script.  
Pushed up. Thanks!

**Comment by covesturtevant** [comment]
Looks great. Thanks @choim-sci !

**Pull request merged**